kidkode/src/main/webapp/script.js

```diff
@@ -80,19 +81,16 @@ function buildQuiz(testQuestions) {
  * hashmap, and increment the points for the corresponding careers in the hashmap.
  */
 function displayResults(testQuestions) {
-  // get answer containers from quiz
+  // get all answer containers
   const answerContainers = quizContainer.querySelectorAll(".answers");
-
   // create empty hashmap for the user's results
   let resultTracker = new Map();
-
   // for each question interate through each carrer in points array...
```

Comment:
very very very nit picky: iterate is spelled wrong

Reply:
Haha, good catch! If only IDEs had spellchecks 😂